.github/workflows/hq-smoke-tests.yml

```diff
@@ -63,77 +77,59 @@ jobs:
           echo ${{env.NOW}}
           pytest -v --rootdir= HQSmokeTests/testCases -n 4 --dist=loadfile --reruns 1 --html=report_${{ matrix.environment }}.html
 
-      - name: Send Pass Email On Deploy Event
-        uses: dawidd6/action-send-mail@v3
-        if: ${{ success() && github.event_name == 'repository_dispatch' }}
+      - name: Set email vars
+        if: always()
+        id: configure_email
+        uses: actions/github-script@v6
+        env:
+            JOB_STATUS: ${{ job.status }}
+            CC_ENV: ${{ matrix.environment }}
         with:
-            # Required mail server address:
-          server_address: smtp.gmail.com
-            # Required mail server port:
-          server_port: 465
-            # Optional (recommended): mail server username:
-          username: ${{secrets.DIMAGIQA_MAIL_USERNAME}}
-            # Optional (recommended) mail server password:
-          password: ${{secrets.DIMAGIQA_MAIL_PASSWORD}}
-           # Required mail subject:
-          subject: ${{ github.event.client_payload.environment }}  PASS - Smoke Tests[#${{github.run_number}}], ${{env.NOW}}
-            # Required recipients' addresses:
-          to: qa@dimagi.com, sameena.shaik@fissionlabs.com
-            # Required sender full name (address can be skipped):
-          from: <${{secrets.DIMAGIQA_MAIL_USERNAME}}>
-            # Optional HTML body read from file:
-          html_body: file:///${{ github.workspace }}/HQSmokeTests/email_pass.html
-            # Optional priority: 'high', 'normal' (default) or 'low'
-          priority: normal
+          script: |
+            const { promises: fs } = require('fs')
```

Comment:
Since I'm far less knowledgable than you on this ground: what does this bit do?

Reply:
The `require` function is a way of loading external modules in Javascript: https://nodejs.org/en/knowledge/getting-started/what-is-require/

The `{promises: fs}` part is a way of unpacking properties from an object. `require` returns the whole module but I only want the `promises.fs` part so instead of having to go `fs.promises.fs` I unpack it. More here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment